### DISCUSSION TITLE
Help with scripting syntax:

### DISCUSSION THREAD

**Discussion by StellariumSentinel** [Stellarium/stellarium#3715] [Q&A] [answered]
Dear all! 
I would need to use the command line script:

ConstellationMgr.setLinesColor(Vec3f(1.0,1.0,0.0));

When I launch this command from the "script console", I get the following error message:

script error: 'Reference'  @ line 11: ReferenceError: Vec3f is not defined

Not knowing the programming language for the script, I don't know the syntax to declare the required variables (var ...). 

Can someone help me? 

Thank you so much!

**Comment by 10110111** [comment, marked as answer]
See Stellarium User Guide, chapter 17. Particularly, section 17.5.2 "The Vec3f problem". This should solve your issue.

**Reply by StellariumSentinel** [reply]
Thanks so much, 
I solved! 
In fact it wasn't difficult at all! :-)
